File: samples/src/main/java/tk/zielony/carbonsamples/feature/HtmlActivity.java
package tk.zielony.carbonsamples.feature;

import android.os.Bundle;
import android.support.annotation.Nullable;

import tk.zielony.carbonsamples.R;
import tk.zielony.carbonsamples.Samples;
import tk.zielony.carbonsamples.SamplesActivity;

public class HtmlActivity extends SamplesActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_html);

        Samples.initToolbar(this, getString(R.string.htmlActivity_title));
    }
}

File: samples/src/main/java/tk/zielony/carbonsamples/feature/HtmlActivity.kt
package tk.zielony.carbonsamples.feature

import android.os.Bundle
import tk.zielony.carbonsamples.SampleAnnotation

import tk.zielony.carbonsamples.R
import tk.zielony.carbonsamples.ThemedActivity

@SampleAnnotation(layoutId = R.layout.activity_html, titleId = R.string.htmlActivity_title)
class HtmlActivity : ThemedActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        initToolbar()
    }
}
